src/instana/instrumentation/pep0249.py

```diff
@@ -2,139 +2,169 @@
 # (c) Copyright Instana Inc. 2018
 
 # This is a wrapper for PEP-0249: Python Database API Specification v2.0
-import opentracing.ext.tags as ext
+from __future__ import annotations
```

Comment:
This import is not necessary.
You can remove it and use the classes listed in the `if TYPE_CHECKING:` block always between doubled-quotes, like this:
```
def _collect_kvs(self, span: "InstanaSpan", sql: str) -> None:
```

Reply:
Done